e2e/order/process.spec.ts

```diff
@@ -0,0 +1,88 @@
+import MercadoPago from '@src/index';
+import { config } from '../e2e.config';
+import { Order } from '@src/clients/order';
+import { CreateOrderData } from '@src/clients/order/create/types';
+import fetch from 'node-fetch';
+
+const mercadoPagoConfig = new MercadoPago({ accessToken: config.access_token });
+
+async function generateToken(): Promise<string> {
+	const cardDetails = {
+		card_number: '5031433215406351',
+		expiration_year: '2025',
+		expiration_month: '12',
+		security_code: '123',
+		cardholder: {
+			name: 'APRO',
+			identification: {
+				type: 'CPF',
+				number: '19119119100'
+			}
+		}
+	};
+
+	const response = await fetch(`https://api.mercadopago.com/v1/card_tokens?public_key=${config.public_key}`, {
+		method: 'POST',
+		headers: {
+			'Content-Type': 'application/json'
+		},
+		body: JSON.stringify(cardDetails)
+	});
+
+	if (!response.ok) {
+		const errorDetails = await response.json();
+		throw new Error(`Failed to generate token: ${response.statusText} - ${JSON.stringify(errorDetails)}`);
+	}
+
+	const data = await response.json();
+	return data.id;
+}
+
+async function createOrder(token: string): Promise<CreateOrderData> {
```

Comment:
Não há necessidade dessa função ser assíncrona, ela apenas retorna um objeto.
```suggestion
function createOrderRequest(token: string): CreateOrderData {
```